src/main/paradox/docs/api/iam/iam-acls-api.md

```diff
@@ -1 +1,206 @@
-# Access Control List
\ No newline at end of file
+# Access Control Lists
+
+Access Control Lists are rooted in the `/v1/acls` collection.
+
+An ACL defines the applications' data access restriction using the following three parameters:
+         
+- permission: the value used to limit application's access to information.
+- identity: the way to identify individuals.
```

Comment:
Update to:
`- identity: a client identity reference, e.g. a certain user, a group, an anonymous user or someone who is authenticated to a certain realm`